src/Tribe/Widget/Manager.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Widgets manager for Tribe plugins.
+ *
+ * @package Tribe\Widget
```

Comment:
Flip the position of `@package` and `@since` here. Add a blank line between `@package` and `@since`.